Archive ACSAC results

principis opened the pull request:
I previously aggregated ACSAC AE results for personal use. I was initially unsure about opening this PR since these results are already stored on the ACSAC website. However, I believe archiving these results on SecArtifacts could be beneficial for broader access and long-term availability. 

These were procedurally generated (I can't provide the scripts at this time, but will try to add them in the future in some form), so please let me know if any changes are necessary.

vahldiek commented:
I think I only have one request, otherwise looks fine. Since they have the curated results lists on their website (which other conferences don't), could you include a link in every result page? Maybe add a sentence at the top saying something to the effect of "results automatically obtained from ACSAC website: <url>".

principis commented:
Good idea! I have updated the results pages to include the link. Let me know if the position or phrasing can be improved. 

I also rearranged the `reproduced` badge after `reusable` as that follows the ACM order, and removed the erroneously added `functional` badge from papers with the `reusable` badge (the reusable badge encompasses the functional badge).

vahldiek commented:
lgtm!
